Source/UI/About/IdentityView.swift

```diff
@@ -10,10 +10,30 @@ import CrashReporting
 import Logger
 import SwiftUI
 
+class IdentityViewBuilder {
+    func build(identity: Identity, botRepository: BotRepository, appController: AppController) -> some View {
```

Comment:
I like what you are trying to do here which I think is prevent the duplicated code `.environmentObject(botRepository).environmentObject(appController)` that was stacking up every time we instantiated a SwiftUI view from UIKit. I think it makes sense to go a little further since we still have some boilerplate when making SwiftUI views. I see two ways we could continue: 
1. give `botRepository` and `appController` default values of `.shared` in this function signature. This means the call site could just be `IdentityViewBuilder().build(identity: identity)` in production code, and we could pass in an alternate Bot or AppController in SwiftUI previews and unit tests as needed.
2. Make a generic view modifier that we could slap on any view we are instantiating from SwiftUI. For instance we could apply it to HashtagView and HomeView, not just IdentityView. It could optionally take a `botRepository` and `appController` just like `IdentityViewBuilder.build()` and could default them to `.shared`. It could look something like this: `IdentityView(identity: identity).injectAppEnvironment()`. In SwiftUI previews and tests it could look more like this: `IdentityView(identity: identity).injectAppEnvironment(botRepository: BotRepository.fake, appController: AppController.mock)`.

Reply:
Implemented both suggestions. I liked the injectAppEnvironment idea!